benchmarks/utils/benchmark_ts.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+# inputs:
+# $1: branch name
+# $2: model yaml files predefined in benchmarks/models_config/
+#     - all: all yaml files in config;
+#     - a list of files separated by comma: bert_multi_gpu.yaml,fastrcnn.yaml
+# $3: (optional) "nightly" trigger branch installation and reports saved in AWS S3
+# Note:
+# - aws cloudwatch or s3 destination can be adjusted based on user's AWS setting.
+# - aws cloudwatch metric-data max size 40kb
+#
+# cmd examples:
+# - cd serve
+# - ./benchmarks/utils/benchmark_ts.sh master all nightly or ./benchmarks/utils/benchmark_ts.sh master all
+# - ./benchmarks/utils/benchmark_ts.sh master bert_multi_gpu.yaml,fastrcnn.yaml nightly or
+#   ./benchmarks/utils/benchmark_ts.sh master bert_multi_gpu.yaml,fastrcnn.yaml
+
+sudo apt install -y apache2-utils
+
+set -ex
+
+BRANCH=$1
+if [ "$BRANCH" == "" ]; then
+    BRANCH=master
+fi
+
+CUR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+echo "CUR_DIR=$CUR_DIR"
+
+if nvidia-smi -L; then
+    hw_type=GPU
+    RUNTIME="--runtime=nvidia"
+    if [ "$2" == "all" ]; then
+        declare -a models=("bert_multi_gpu.yaml" "fastrcnn.yaml" "mnist.yaml" "vgg16.yaml" )
+    else
+        IFS="," read -a models <<< $2
+    fi
+
+    echo "switch to CUDA 10.2"
+    sudo rm -rf /usr/local/cuda
+    sudo ln -s /usr/local/cuda-10.2 /usr/local/cuda
+    export PATH=/usr/local/cuda-10.2/bin${PATH:+:${PATH}}$
+    export LD_LIBRARY_PATH=/usr/local/cuda-10.2/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
+else
+    hw_type=CPU
+    if [ "$2" == "all" ]; then
+        declare -a models=("bert_cpu.yaml" "fastrcnn.yaml" "mnist.yaml" "vgg16.yaml" )
+    else
+        IFS="," read -a models <<< $2
+    fi
+fi
+
+# directory to store execution log
+rm -rf /tmp/ts_benchmark/
+rm -rf /tmp/benchmark
+mkdir -p /tmp/benchmark
+
+# clone TorchServe 
+#if [ ! -d "serve" ]; then
+#    git clone --quie https://github.com/pytorch/serve.git > /dev/null
+#fi
+
+#cd serve || exit 1
+
+# install TorchServe
+if [[ "$3" == "nightly" ]]; then
```

Comment:
you can leverage `pip install torchserve-nightly` now and remove most the installation code

Reply:
yes, it is in plan.